src/components/History/NetworkUpgradeSummary.tsx

```diff
@@ -24,11 +27,8 @@ interface IProps {
 
 const NetworkUpgradeSummary: React.FC<IProps> = ({ name }) => {
   const [formattedUTC, setFormattedUTC] = useState("")
-
   const { locale } = useRouter()
-
-  // const localeForStatsBoxNumbers = getLocaleForNumberFormat(language as Lang)
-  const localeForStatsBoxNumbers = "en"
+  const localeForStatsBoxNumbers = DEFAULT_LOCALE
```

Comment:
We have `getLocaleForNumberFormat` already migrated, located in `@/lib/utils/translations`

If we add 

```
import { getLocaleForNumberFormat } from "@/lib/utils/translations"
```

at the top, we can just uncomment L30 and change `language` to `locale`

```
const localeForStatsBoxNumbers = getLocaleForNumberFormat(locale as Lang)
```

Reply:
Right, fixed in last commit, thanks!